app/views/static/home.html.erb

```diff
@@ -1,11 +1,187 @@
+          </div>
+          <div calss="form-group">
+            <div class="hr-label">
+              <span class="hr-label__text">or</span>
+            </div>
+          </div>
+          <div class="form-group">
+            <button type="submit" class="btn btn-default btn-block">Signup for Free with Facebook</button>
+          </div>
+        </div>
+        <div class="hidden-xs col-xs-12 col-sm-5">
+          <h3 class="signup-text">Sign up</h3>
+          <form class="form-horizontal signup-form cover-signup-form">
+            <div class="form-group">
+              <div class="col-xs-12">
+                <input class="form-control" placeholder="Full Name">
+              </div>
+            </div>
+            <div class="form-group">
+              <div class="col-xs-12 col-md-6">
+                <input type="email" class="form-control" placeholder="Email">
+              </div>
+              <div class="col-xs-12 col-md-6">
+                <input type="password" class="form-control" placeholder="Password">
+              </div>
+            </div>
+            <div class="form-group">
+              <div class="col-xs-12">
+                <div class="checkbox">
+                  <label>
+                    <input type="checkbox"> I agree to the <a class="privacy-link"href="#">Erudition Terms</a>
+                  </label>
+                </div>
+              </div>
+            </div>
+            <div class="form-group">
+              <div class="col-xs-12">
+                <button type="submit" class="btn btn-default btn-block">Signup for Free</button>
+              </div>
+            </div>
+            <div calss="form-group">
+              <div class="col-xs-12 hr-label">
+                <span class="hr-label__text">or</span>
+              </div>
+            </div>
+            <div class="form-group">
+              <div class="col-xs-12">
+                <button type="submit" class="btn btn-default btn-block">Signup for Free with Facebook</button>
+              </div>
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <section class="home-section home-section--gray">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12 col-sm-5">
+          <div class="home-section__thumb">
+            <%= image_tag('static/home-screenshot-01.jpg') %>
+          </div>
+        </div>
+        <div class="col-xs-12 col-sm-7">
+          <h2 class="home-section__text">
+            <small>Section One</small>
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat
+          </h2>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="home-section">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12 col-sm-5 col-sm-push-7">
+          <div class="home-section__thumb">
+            <%= image_tag('static/home-screenshot-02.jpg') %>
```

Comment:
In Ruby, parentheses are often times optional. That weirded me out at first, but it's pretty typical to leave the parentheses out in this case `<%= image_tag 'static/home-screenshot-02.jpg' %>` at least that's what I've seen in most the code I've maintained.

I looked up some style guides so I could just confirm. AirBnB has a Ruby style guide, hah, we could refer to this for consistency (we just LOOVE airbnb). https://github.com/airbnb/ruby#method-calls In this case the 5th bullet point describes this scenario. They don't have a strong opinion. I normally wouldn't use parentheses here but mostly I am concerned about consistency, so I'll leave it up to you. Let me know!